src/app/_components/RichText/index.module.scss

```diff
@@ -1,5 +1,5 @@
 .richText {
-  text-align: center; // hack but whatever
+  // text-align: center; // hack but whatever
```

Comment:
just wondering if this is something we want to do here? If possible, could we leave it and just override with tailwind css styling?